mlflow/pytorch/__init__.py

```diff
@@ -591,6 +680,33 @@ def _load_model(path, **kwargs):
             return torch.jit.load(model_path)
 
 
+@experimental
+def load_state_dict(model_uri):
+    """
+    Load a state dict from a local file or a run.
+
+    :param model_uri: The location, in URI format, of the MLflow model, for example:
+
+                    - ``/Users/me/path/to/local/model``
+                    - ``relative/path/to/local/model``
+                    - ``s3://my_bucket/path/to/model``
+                    - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+                    - ``models:/<model_name>/<model_version>``
+                    - ``models:/<model_name>/<stage>``
+
+                    For more information about supported URI schemes, see
+                    `Referencing Artifacts <https://www.mlflow.org/docs/latest/concepts.html#
+                    artifact-locations>`_.
+    :return: A pytorch model state dict of type OrderedDictionary
```

Comment:
Is the return tyep alwasy `OrderedDictionary`?

```python
torch.save({"model": model.state_dict(), "other_item": ...})
type(state_dict = torch.load(model_path))  # -> ???
```

Reply:
Yes. When i save the state dict using `torch.save` and load the state dict back, the type is `OrderedDictionary`

For example:
```
>>> 
>>> import torch
>>> 
>>> 
>>> type(torch.load("iris.pt"))
<class 'collections.OrderedDict'>
>>> 
```

Reply:
How did you save `iris.pt`?

```python
# torch.save(model.state_dict(), ...)  # not this
torch.save({"model": model.sttate_dict(), ...})
```

Reply:
Got it.. In this case, it would be of type `dict`. Removed `Ordered Dictionary` from docstring